### DISCUSSION TITLE
Great Tool

### DISCUSSION THREAD

**Discussion by lu-kas** [executablebooks/meta#226] [Thanks]
Dear developers, especially @choldgraf, 

I am very much enjoying the jupyterbook project! I used the old (legacy) version for a lecture

web: https://io.cce.uni-wuppertal.de/lecture_inginf/Overview/uebersicht
repo: https://github.com/lu-kas/Vorlesung_Ingenieurinformatik

I am currently working on a new lecture using the new jupyterbook version and it is a huge improvement to the previous one. Many of my students enjoyed the way jupyterbook presents the contents. They will especially like the new PDF builder, as this became way better in the new version.  

Thanks for your effort.

**Comment by choldgraf** [comment]
Thanks @lu-kas ! Though no need to specifically thank me, everybody in the EBP community is putting a lot of work into this project :-)
